ush/python/pygfs/task/jedi.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from pprint import pformat
+from typing import List, Dict, Any, Optional
+from jcb import render
+from wxflow import (AttrDict,
+                    chdir, rm_p,
+                    parse_j2yaml, save_as_yaml,
+                    logit,
+                    Task,
+                    Executable,
+                    WorkflowException)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class JEDI(Task):
```

Comment:
This looks like a class hard-wired for the analysis step, which I think is fine but the name should probably reflect this. `JEDIAnalysis`? or just keep `Analysis`?
I think @CoryMartin-NOAA (or maybe that was you @DavidNew-NOAA ?) started a b-matrix base class as well. We probably want to keep these separate.

Reply:
The intention is for this class to be used anytime a JEDI application is run in general, not just for running an analysis